apkid/rules/elf/packers.yara

```diff
@@ -289,3 +289,28 @@ rule upx_unknown_version_unmodified : packer
     upx_unmodified and
     not upx_compressed_apk
 }
+
+rule ollvm_obfuscator : obfuscator
+{
+  meta:
+    description = "Obfuscator-LLVM native"
```

Comment:
The 'native' here should be implied by the file name or type mapped to this detection. For that reason, and to make consistent with other names, how about just "Obfuscator-LLVM".

Also, could you add a comment above the description with a URL to the best reference site for this. And can you do the same for Arxan? This way, if someone wants to know "WTF IS ARXAN?" they can look at the rules and find the site.

Reply:
- Removed "native" and "dalvik" following the style.
- Added tag "info="http://url.com" for obfuscators.